### ISSUE TITLE
Appropriate notices should be provided to users about search/list delays

### ISSUE THREAD

**Issue by skylerbunny** [internetarchive/openlibrary#425]
As a new user to OpenLibrary, I might reasonably expect a change I make (such as a new book being added to an author) immediately showing up when I subsequently search for that author as proof of my accomplishment. Of course as I examine the code here, I know this isn't true; the solr process runs to make changes where works and authors are searchable every 15 minutes. For all intents and purposes, a new author (if applicable) or a new work is only browsable for the first few minutes by going directly to its OL ID.

_The problem is, a user new to the site will not know this._ Many users who see a delay like this and are not cautioned in some way may be likely to try to create duplicate books, or to create duplicate new authors (each of which have the same single newly added book) because they'll believe that their request 'didn't go through'.

OpenStreetMap handles this by giving a notice when someone saves a change in iD:
Your changes should appear in the "Standard" layer in a few minutes. Other layers, and certain features, may take longer. [Details](https://wiki.openstreetmap.org/wiki/FAQ#I_have_just_made_some_changes_to_the_map._How_do_I_get_to_see_my_changes.3F)

I suggest something like this, which will help solve both the problem of 'understanding that it takes a few minutes to see the change on the live site', but also 'wanting to confirm the creation one has just done', as [here](https://github.com/internetarchive/openlibrary/blob/f8092840a77c7479a352fd83fd068d340f97d3e3/openlibrary/templates/messages.html#L8):

'Thank you very much for adding this [LINK]new book[/LINK]! It should appear in title searches and under [LINK]the author[/LINK] in a few minutes.

In the meantime, do you have more information you can add?'

**Comment by guyjeangilles** [comment]
@skylerbunny While there's no notice that I've encountered while editing, I haven't encountered the delayed indexing, so I'm proposing to close this issue. @mekarpeles I'm not sure who should be assignee so I'm defaulting to you. Note: being assignee doesn't necessarily mean you are the one doing the work for the issue but responsible for collecting information needed to devise a solution. From the wiki:

>At submission and prior to triage, the assigned owner is not necessarily the person who will fix the issue (it is not necessarily even established, at that point, if or when the issue will be fixed at all), but rather they are the person who will do as much or as little as needed to handle the issue (asking questions, soliciting input, establishing and updating the priority, checking if it is a duplicate, etc).
>
>Once an issue is labeled State: Work In Progress, the owner is the individual doing the work, or leading/coordinating the group that is doing the work.

I've added labels per the context. Let me know your thoughts.

**Comment by LeadSongDog** [comment]
@guyjeangilles I don’t agree. When adding the first work by an author, one creates a new author record. It gets a new olid. Until indexed, that new record is only accessible via that olid URL, meaning that author record cannot be added to other works. This is a real problem because it leads to the creation of duplicate author records. It needs fixing, or at a bare minimum an explanation to users.

**Comment by guyjeangilles** [comment]
Now that I've specifically added the first work to an author, I can see how a user would repeatedly add authors until one of them indexed. I agree with you @LeadSongDog
